operator/pkg/utils/superusers.go

```diff
@@ -0,0 +1,55 @@
+package utils
```

Comment:
I really don't like "kitchen sink" packages as it's very difficult to discover what's in them and reason about where things should go.

As we can't put this into the configwatcher package due to it being internal, could we instead put it into the syncclusterconfig package? You can put it in a utils.go file there :P

Reply:
Sure